Cargo.toml

```diff
@@ -21,17 +22,18 @@ homepage = "https://github.com/stainless-steel/special"
 repository = "https://github.com/stainless-steel/special"
 readme = "README.md"
 categories = ["algorithms", "no-std::no-alloc", "science"]
-keywords = ["beta", "error", "lambert-w", "gamma"]
+keywords = ["beta", "elliptic", "error", "lambert-w", "gamma"]
 
 [features]
 default = ["no_std"]
 
 # If one of the two features is disabled, the other one must be enabled. If both
 # features are enabled, `std` takes precedence over `no_std`.
-no_std = ["lambert_w/libm"]
+no_std = ["lambert_w/libm", "ellip/no_std"]
```

Comment:
Now that I understood what is going on inside, let us remove it here and enable permanently in the dependencies section.